.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
--   repo: https://github.com/ambv/black
+-   repo: https://github.com/psf/black
```

Comment:
dang, moved to the PSF eh?  Must be official :grin:

Reply:
haha, yup its a thing now :D